backend/curriculum_tracking/tests/test_command_export_project_urls.py

```diff
@@ -0,0 +1,19 @@
+from core.tests.factories import TeamFactory, UserFactory
+from curriculum_tracking.tests.factories import ProjectContentItemFactory
+from django.test import TestCase
+from curriculum_tracking import models
+from curriculum_tracking.management.commands.export_project_urls import get_email_and_url
+
+class TestCommandExportProject(TestCase):
+
+    def test_project_not_submitted(self):
+        self.team = TeamFactory()
```

Comment:
Why are you using self? Are you using `team` outside this method?

Try not to let information leak out a function unless you need it to.